src/main/java/com/es/phoneshop/model/product/ProductNotFoundException.java

```diff
@@ -0,0 +1,11 @@
+package com.es.phoneshop.model.product;
+
+public class ProductNotFoundException extends RuntimeException
```

Comment:
Don't make your own throwables without necessarity. Make use of NoSuchElementException or smth like that